enterprise/metrics/src/main/java/org/neo4j/metrics/MetricsSettings.java

```diff
@@ -130,6 +135,15 @@
                   "the CSV files." )
     public static Setting<Duration> csvInterval = setting( "metrics.csv.interval", DURATION, "3s" );
 
+    @Description( "The file size in bytes at which the csv files will auto-rotate. If set to zero then no " +
+            "rotation will occur. Accepts a binary suffix `k`, `m` or `g`." )
+    public static final Setting<Long> csvRotationThreshold = buildSetting( "metrics.csv.rotation.size",
+            BYTES, "20m" ).constraint( range( 0L, Long.MAX_VALUE ) ).build();
+
+    @Description( "Maximum number of history files for the csv files." )
+    public static final Setting<Integer> csvMaxArchives = buildSetting( "metrics.csv.rotation.keep_number",
```

Comment:
Roughly how long will these defaults keep metrics before being rotated out completely?

A quick calculation, assuming 100 bytes per logged line, yields roughly 7 days worth of data in 20MB. So ~50 days worth of data in there.

Even given that I see no reason for having the rotation threshold that low. 20MB is not that much, I think might as well be 100 or more imho. Most editors can handle 100 MB or a couple, can't they?

Reply:
sure, let's do them 100m by default for now if that will be too big since we can have a lot of metrics files and all of them will consume space we can change it later on